src/registered.js

```diff
@@ -71,14 +59,48 @@ module.exports = async function (request, h) {
     }],
   };
 
-  fetch(config.apiURL + '/publishers', {
-    method: 'POST',
-    body: JSON.stringify(apiPayload),
-    headers: {'Content-Type': 'application/json', 'Authorization': 'Bearer ' + apiPasetoKey}
-  }).then(res => {
-    console.log(res.json());
-  });
+  const apiURL = config.apiURL.replace(/\/$/, '');
+
+  try {
+    const res = await fetch(`${apiURL}/publishers`, {
+      method: 'POST',
+      body: JSON.stringify(apiPayload),
+      headers: {'Content-Type': 'application/json', 'Authorization': `Bearer ${apiPasetoKey}`},
+    });
+
+    const data = await res.json();
+
+    if (!res.ok) {
+      throw `errore imprevisto nel salvataggio, riprovare più tardi (${data?.title}, ${data?.detail})`;
+    }
 
+    db.get('registrati')
+      .push({
+        timestamp: new Date().toJSON(),
+        referente: referente,
+        refTel: refTel,
+        ipa: ipa,
+        url: url,
+        pec: pec,
+      })
+      .write();
```

Comment:
I have some concern about, this could potentially lead to out of sync between databases.

Reply:
How? The write to the local file should happen only when the API returned a successful status

Reply:
Ops, my bad...